portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,9 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+function addQuoteToDom(quote) {
```

Comment:
You must have a function that matches `addRandomQuote()` specified in the HTML.

For example, you can copy `addRandomGreeting` and modify it to get a quote:

```
function addRandomQuote() {
  const quote =
      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
  // Pick a random quote.
  const quote = greetings[Math.floor(Math.random() * greetings.length)];
  // Add it to the page.
  const quoteContainer = document.getElementById('quote-container');
  quoteContainer.innerText = quote